tests/e2e/actions.go

```diff
@@ -1770,13 +1809,15 @@ type DoublesignSlashAction struct {
 
 func (tr TestConfig) invokeDoublesignSlash(
 	action DoublesignSlashAction,
+	target ExecutionTarget,
 	verbose bool,
 ) {
 	if !tr.useCometmock {
 		chainConfig := tr.chainConfigs[action.Chain]
-		//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
-		bz, err := exec.Command("docker", "exec", tr.containerConfig.InstanceName, "/bin/bash",
-			"/testnet-scripts/cause-doublesign.sh", chainConfig.BinaryName, string(action.Validator),
+		isConsumer := false
```

Comment:
inline this? it's used only once, right below

Reply:
unless you like it better like this for clarity, I can see an argument for that, too